src/modules/window-notifier.js

```diff
@@ -54,12 +54,8 @@
 	};
 
 	var requestScrollTop = function (key, data) {
-		if (data && data.animated) {
-
-		} else {
-			win.scrollTop(0);
-			scrollHandler();
-		}
+		win.scrollTop(0);
+		scrollHandler();
```

Comment:
This changes the actual behaviour. Are we just we want it ? I think it should be:

```js
if (!data || !data.animated) {
    win.scrollTop(0);
    scrollHandler();
}
```

So we won't have the warning about the empty if statement.

Reply:
@PascalPiche ping!